StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentElementTest.swift

```diff
@@ -188,6 +188,9 @@ class EmbeddedPaymentElementTest: XCTestCase {
     }
 
     func testConfirmHandlesInflightUpdateThatFails() async throws {
+        let paymentIntentConfig = EmbeddedPaymentElement.IntentConfiguration(mode: .payment(amount: 1000, currency: "USD"), paymentMethodTypes: ["card", "cashapp"]) { _, _, _ in
+            XCTFail("Confirm handler should not be called.")
```

Comment:
let's just add this XCTFail to the `paymentIntentConfig` property and not make a new one?

Reply:
Yeah that's a better idea, I added the XCTFail to all the other confirm handlers that should not be called as well.